_data/leaderboard.yml

```diff
@@ -0,0 +1,24 @@
+- Fellow: Helen Du
```

Comment:
It would be best to follow the same naming convention as the other files in `/_data` (kebab-case).

Reply:
changes to kebab-case

Reply:
Excellent!